src/project/internal/projectactionscontroller.cpp

```diff
@@ -359,7 +360,9 @@ bool ProjectActionsController::closeOpenedProject(bool quitApp)
         globalContext()->setCurrentProject(nullptr);
 
         if (quitApp) {
-            dispatcher()->dispatch("quit", actions::ActionData::make_arg1<bool>(false));
+            async::Async::call(this, [this](){
+                dispatcher()->dispatch("quit", actions::ActionData::make_arg1<bool>(false));
```

Comment:
Maybe `async` needs to be added to the `quit` action handler? Or is the `async` needed only in this place? Then I think it's better to add a comment why

Reply:
This is only needed here since we control the life cycle of the method(m_isProjectClosing) and cannot call such methods synchronously